Restore for Extra area step 2

mahotilo opened the pull request.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/Typesetter/Typesetter/inspections/037b0e0a-015b-4431-ac27-97493ae89856) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/Typesetter/Typesetter/inspections/037b0e0a-015b-4431-ac27-97493ae89856) completed: **No new issues**

mahotilo commented:
It is for https://github.com/Typesetter/Typesetter/issues/530 again.
![image](https://user-images.githubusercontent.com/14929385/72707594-72b08f80-3b69-11ea-90b4-c2b8cf742545.png)


I have changed 'cmds' value to 'Redirect' and added Restore button to Extra Content » Ares_name page. Now it all works fine in Extra.php, but I don't succeed with manage_sections.js
1. Lack of gplang.restore. I have to temporary use gplang.ca instead
2. I don't know how to stay on the same edited page after clicking the restore(cancel) button. Command works but opens Administration » Extra Content page and started an endless circle of Oops'es.

juek commented:
> Lack of gplang.restore. I have to temporary use gplang.ca instead

We should eventually call it 'Dismiss Draft'.
